### DISCUSSION TITLE
How to add users to a project?

### DISCUSSION THREAD

**Discussion by theranajayant** [appwrite/appwrite#3939] [Q&A] [answered]
### 👟 Reproduction steps

1. On a fresh installation 
2. created a project
3. created a user account in the project 
4. and tried logging in with the new account using a browser, but the authentication fails and says "invalid credentials". 

![Screen Shot 2022-09-12 at 9 18 20 AM](https://user-images.githubusercontent.com/12654266/189570427-49d9f509-574c-46e6-b8e2-ea0985eaf83f.png)

### 👍 Expected behavior

When I create user with creds, 
I should be able to login with the new account I just created and be able to manage the resouresresourcesresources resources

### 👎 Actual Behavior

Throws error saying "wrong credentials". 
I tried clicking "verify mail" as well but nothing worked. 

![Screen Shot 2022-09-12 at 9 18 20 AM](https://user-images.githubusercontent.com/12654266/189570427-49d9f509-574c-46e6-b8e2-ea0985eaf83f.png)

### 🎲 Appwrite version

Version 0.15.x

### 💻 Operating system

MacOS

### 🧱 Your Environment

not customized, no SMTP, tested on 
1. digital ocean using one-click install and 
2. in the local machine in MacOS one-click

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by stnguyen90** [comment, marked as answer]
@theranajayant, the user you created in the project is an end user meant to log into your app, rather than a user meant to log into the appwrite console. To add a user for the appwrite console, go to the project settings > members > and invite a new member.
